### ISSUE TITLE
Optimize Symmetrization

### ISSUE THREAD

**Issue by nickbattam-tessella** [pace-neutrons/Horace#24]
This tool must be optimized for multi-thread and multi-core execution.

Identified by Alex as benefiting from parallelization

**Comment by nickbattam-tessella** [comment]
Depends on #55

**Comment by oerc0122** [comment]
#990 #994 deals with this

**Comment by cmarooney-stfc** [comment]
@oerc0122 bring up to date

**Comment by cmarooney-stfc** [comment]
still to be parallelised - superseded by #1300
